<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Decrypt</class>
 <widget class="QDialog" name="Decrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="driveBox"/>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QPushButton" name="mountConfirm">
     <property name="text">
      <string>Mount...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonFullDe">
     <property name="text">
      <string>Full Decryption</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonHandoff">
     <property name="text">
      <string>Add Users to Handoff...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="buttonEnroll">
     <property name="text">
      <string>Enroll New User...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="buttonDeCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
